crates/hyperswitch_connectors/src/types.rs

```diff
@@ -71,3 +79,35 @@ pub type FrmFulfillmentType =
     dyn ConnectorIntegration<Fulfillment, FraudCheckFulfillmentData, FraudCheckResponseData>;
 pub type FrmCheckoutRouterData =
     RouterData<Checkout, FraudCheckCheckoutData, FraudCheckResponseData>;
+
+pub type PreAuthNRouterData =
```

Comment:
```suggestion
pub(crate) type PreAuthNRouterData =
```
Mark all these types pub(crate)